src/rtp/srtp.rs

```diff
@@ -273,12 +465,15 @@ impl SrtpKey {
     }
 
     fn derive(&self, label: u8, out: &mut [u8]) {
+        // AEC-CM (128 bits) defined in RFC3711
+        assert!(ML == 16, "Only valid for 128 bit master keys");
+        assert!(SL <= 14, "Only valid for 128 bit master keys");
```

Comment:
Should really be const asserts, but then the ML and SL generics here are a bit overkill in the first place perhaps

Reply:
Do we ever clone/copy them?

Reply:
Don't think so